retrievalmarket/README.md

```diff
@@ -0,0 +1,274 @@
+# How to use the retrievalmarket module
```

Comment:
- This is long and may benefit from a Table of Contents ([example](https://github.com/RichardLitt/standard-readme))
- Let's separate Purpose/Background from Operation/Usage